src/OrchardCore/OrchardCore.Infrastructure.Abstractions/OrchardCore.Infrastructure.Abstractions.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <ProjectReference Include="..\OrchardCore.Abstractions\OrchardCore.Abstractions.csproj" />
     <ProjectReference Include="..\OrchardCore.Data.Abstractions\OrchardCore.Data.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.ResourceManagement.Abstractions\OrchardCore.ResourceManagement.Abstractions.csproj" />
```

Comment:
Seems strange - refe is added for the project but don't see changes for reference used in code of this project?

Reply:
`ISite` belongs to `OrchardCore.Infrasturucture.Abstraction` which contains reference to `ResourceDebugMode` which belongs to  `OrchardCore.Infrasturucture.Abstraction`